packages/api/cms-api/src/user-permissions/decorators/required-permission.decorator.ts

```diff
@@ -0,0 +1,29 @@
+import { EntityClass, EntityManager } from "@mikro-orm/core";
+import { CustomDecorator, SetMetadata } from "@nestjs/common";
+import { Request } from "express";
+
+import { CurrentUser } from "../dto/current-user";
+import { ContentScope } from "../interfaces/content-scope.interface";
+import { Permission } from "../interfaces/user-permission.interface";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type RequiredPermissionArgs<ArgsType = any> = {
```

Comment:
this type is not yet used, I guess you'll need it soon tough

Reply:
Yes, the annotations and the discussion about it will follow in the next PR.